lib/potluck.rb

```diff
@@ -0,0 +1,26 @@
+class Potluck 
+  attr_reader :date, 
+              :new_dish,
+              :dishes
+            
+
+  def initialize(date)
+    @date = date 
+    @dishes = []
+    @new_dish = new_dish
```

Comment:
Be careful here!  ruby does not know what `new_dish` is when you are initializing the class!